### ISSUE TITLE
关于训练过程中计算的loss_ratio是一四维张量不是标量，在计算“l_total += loss_ratio”时出现报错

### ISSUE THREAD

**Issue by mhmhh404** [icandle/CAMixerSR#44]
您好！感谢您的优秀工作，我这里已可以使用预训练权重对图像进行推理，但我在进行训练时，在CAMixerSR_model.py的147行代码“l_total += loss_ratio”出现报错，原因时l_total是一个标量，而loss_ratio是一个四维张量，不知道是什么原因导致。
找到loss_ratio值的计算代码，是在CAMixerSR_arch.py的419行，应该是“2*self.ratio*(torch.mean(torch.cat(decision,dim=1),dim=(0,1))-0.5)**2”中decision变量的维度不对，请问如何解决？

![Image](https://github.com/user-attachments/assets/dc29f3a2-f614-4423-abce-932338e6196d)

**Comment by mhmhh404** [comment]
loss_ratio是一个1*4的张量

**Comment by icandle** [comment]
Hi， 如果维度不对，直接将decision取平均和0.5做MSE也可以

**Comment by mhmhh404** [comment]
受到，感谢回复！
